translations/translation_hu.xml

```diff
@@ -129,13 +130,13 @@
 		<text name="gui_ad_findDriver_tooltip"						text="Keresse meg a legközelebbi sofőrt (figyelmen kívül hagyja a távolságot)" />
 		<text name="gui_ad_gui_scale"								text="Az AutoDrive HUD grafikus felhasználói felületének méretezése" />
 		<text name="gui_ad_gui_scale_tooltip"						text="Az Alapértelmezett beállítás kiválasztásakor a játék beállításai irányadók" />
-		<text name="gui_ad_notifications"							text="Értesítés" />
+		<text name="gui_ad_notifications"							text="Értesítés megjelenítési ideje" />
 		<text name="gui_ad_notifications_text_1"					text="Tiltva" />
-		<text name="gui_ad_notifications_text_2"					text="Rövid" />
-		<text name="gui_ad_notifications_text_3"					text="Alapértelmezett" />
-		<text name="gui_ad_notifications_text_4"					text="Hosszú" />
-		<text name="gui_ad_notifications_text_5"					text="Leghosszabb" />
-		<text name="gui_ad_notifications_text_6"					text="Végtelen" />
+		<text name="gui_ad_notifications_text_2"					text="2,5s" />
```

Comment:
This should not use absolute times. Messages have different notification durations, and this setting just changes the display duration relative to the default duration.